jdaviz/configs/cubeviz/plugins/sonify_data/sonify_data.py

```diff
@@ -54,11 +54,11 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self._plugin_description = 'Sonify a data cube'
         self.docs_description = 'Sonify a data cube using the Strauss package.'
-        if not self.has_strauss or len(sd.default.device) < 1:
+        if not self.has_strauss or len(list(sd.default.device)) < 2:
```

Comment:
problem with this is that `sd.default.device` will always be 2 I think (`[<input device index>,<output device index>]`), instead can check that the value < 0 as this means no devices I think

Reply:
this will also fail if `sd` undefined in the no-strauss support install case - think the implementation here should work: https://github.com/javerbukh/jdaviz/pull/23/files#diff-844f042745fb5ecac2f1885915c486386a58260415a4232074887f1614f424ea